Cargo.toml

```diff
@@ -21,3 +21,9 @@ tokio = { version = "1.45.0", features = ["full"] }
 tokio-util = { version = "0.7.11", features = ["io"] }
 tracing = "0.1.41"
 tracing-subscriber = "0.3.19"
+# compression
+flate2 = "1.1.2"
+zstd = "0.13"
+lz4 = "1.24"
+snap = "1.1"
+brotli = "8.0.1"
```

Comment:
to use [async_compression](https://docs.rs/async-compression/latest/async_compression/)